### PR TITLE
BAPL-839 (for upstream master): Updated Template Key help text in GDT and guided rule designers

### PR DESCRIPTION
**Pull request by sterobin**
Details in [JIRA](https://issues.jboss.org/browse/BAPL-839).

### CODE REVIEW

**Comment by manstis** [comment]
@sterobin You'll need to add ```String TemplateKeyTip();``` to [this](https://github.com/kiegroup/drools-wb/blob/master/drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/resources/i18n/Constants.java) file.

**Comment by sterobin** [comment]
@manstis, thanks. Done. We'll see how she builds.

File: drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/resources/i18n/Constants.java [hunk 1]
```diff
@@ -278,6 +278,8 @@ String Append0ToList1( String varName,
 
     String TemplateKey();
 
+    String TemplateKeyTip();
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/resources/i18n/Constants.java]: :+1:

**Comment by manstis** [comment]
@sterobin Let's see what Jenkins makes of your programming prowess!

**Comment by sterobin** [comment]
@jomarko , my vote is to merge and submit a JIRA to widen the tooltip. It's a problem with all of the help icons in the window anyway. Plus it's better to have a hard-to-read help tip that essential, then not have it at all (in this case).

**Pull request merged**